test/CMakeLists.txt

```diff
@@ -24,4 +24,8 @@ add_executable(soltest ${sources} ${headers}
 )
 target_link_libraries(soltest PRIVATE libsolc solidity lll evmasm devcore ${Boost_UNIT_TEST_FRAMEWORK_LIBRARIES})
 
+if (NOT Boost_USE_STATIC_LIBS)
```

Comment:
Where is this variable defined? It should be a _cache_ variable if you want to change the value with command line options.

Reply:
https://github.com/ethereum/solidity/blob/8b2d630275fb29d2915af76f608be4d4e422907f/cmake/EthDependencies.cmake#L39